locales/es/es.json

```diff
@@ -551,10 +552,10 @@
     "Switzerland": "Suiza",
     "Syrian Arab Republic": "Siria",
     "Taiwan": "Taiwán",
-    "Taiwan, Province of China": "Taiwan, Province of China",
+    "Taiwan, Province of China": "Taiwan, provincia de China",
```

Comment:
Taiwán ?

Reply:
[https://github.com/Laravel-Lang/lang/blob/master/source/en.json#L553](https://github.com/Laravel-Lang/lang/blob/master/source/en.json#L553) is dufplicate from `source/en.json` not `es.json`